loadgenerator/loadgen

```diff
@@ -400,6 +399,45 @@ def _check_prereqs():
     except subprocess.CalledProcessError:
         return False
 
+#####################################
+# CLOUD RUN LOAD GENERATOR COMMANDS #
+#####################################
+
+def run_auto(args):
+    """Create automatic load generator and Web UI for normal traffic pattern.
+    Use Cloud Run to host the container and serve the Locust Web UI.
+    args.target_address -- the IP address to send load (frontend)
+    """
+    # _setup_cloud_run()
+    name = _generate_job_name()
+    logger.info('Creating load generation job {0}...'.format(name))
+
+    if not _is_address_accessible(args.target_address):
+        logger.error('Could not access target address {0}'.format(args.target_address))
+        print('Please double check the target address and try again.')
+        return
+
+    command = (
+        'gcloud run deploy {0}'
+        ' --image gcr.io/stackdriver-sandbox-230822/sandbox/loadgenerator:gz'
+        ' --update-env-vars="FRONTEND_ADDR={1}"'
+        ' --allow-unauthenticated'
```

Comment:
I get a prompt asking for a target platform. You should be able to set the value as an argument here (we want fully managed)

Reply:
You may also want to set the region, but that might also make sense to keep manual

Reply:
:O they should be automatically set in the _setup_cloud_run() function?

Reply:
Oh oops it got commented out somehow - I'll put it back in with the next push